devops/ext/docker/python3/xenial/Dockerfile.0.1.0

```diff
@@ -1,4 +1,4 @@
-FROM evernym/base:0.2.0-xenial
+FROM sovrin/base:0.2.0-xenial
```

Comment:
As far as I am aware, sovrin does not have it's own custom base images for ubuntu xenial. This seems to be a bigger issue that just this instance though, since there are a number of dockerfiles here that simply have the `evernym` in them replaced with `sovrin`, which assumes that sovrin has the same custom images that EV has. 

Potential solutions:
* Replace custom images with generic images like `FROM ubuntu:16.04`
* If these dockerfiles are non-trivially dependent on their base images, then we should consider creating a set of custom base images for SF

Reply:
old dockerfiles should be cleaned up. automation does make base images in ecr repos.

Reply:
This is an old dockerfile. It does not need to be updated @stevengubler.